### ISSUE TITLE
map ConstructSignature

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#24]
From #19.

**Comment by alfonsogarciacaro** [comment]
Following the idea of having only interfaces and separate the _instance_ and _static_ parts of a type, my intention was to follow a convention where constructors would be `Create` methods in the static part. For example:

```js
class Foo {
   constructor();
   myProperty: number;
   myMethod(i: number): void;
   static myStaticMethods(s: string): number;
}
```

```fsharp
type Foo =
    abstract myProperty: float with get, set
    abstract myMethod: float -> unit

type FooStatic =
    [<Emit("new $0($1...)">]
    abstract Create: unit->Foo
    abstract myStaticMethods: string -> float

type IExports =
    abstract Foo: FooStatic
```

**Comment by ctaggart** [comment]
fixed by #38
